arm/Microsoft.SignalRService/.bicep/nested_rbac.bicep

```diff
@@ -0,0 +1,61 @@
+@sys.description('Required. The IDs of the principals to assign the role to.')
+param principalIds array
```

Comment:
Please add both an AzureDevOps & GitHub module pipeline in the same vein of the other pipelines. It's really more a copy-paste effort with minor adjustments to the triggers.

Reply:
I don't really know what you mean, can you please elaborate?

Reply:
My bad. As it is a new module, there are currently no Azure DevOps & Github pipelines in the `.azuredevops\modulePipelines` & `.github\workflows` folder respectively.

So the ask would be to add a `ms.signalrservice.webpubsub.yml` to each folder respectively. Their content can be copied from any of the other module test pipelines and just require you to update each pipeline's trigger & pipeline variables.

Reply:
I hope that helped

Reply:
Got that, thanks for the explanation

Reply:
I haven't worked with Azure DevOps in a while, except for projects migrating from DevOps to GitHub, but it shouldn't take that long to just copy and replace the workflows

Reply:
Sounds good. If you use neither the GitHub Workflows nor Azure DevOps pipelines I take it you don't have the fork completly set up, right? If so, I'd suggest to create a PR-Branch for you to which we point this Pull-Request (instead of main) and I'll run the pipeline(s) in our environment.

Reply:
Let me know which works best for you

Reply:
Same question for the other PR's I guess

Reply:
I have not setup a environment with GitHub workflows and Azure (DevOps) pipelines. I just forked, created my separate branch and a PR from there.
If there is a chance to run the tests on your side that would be very appreciated!

Reply:
I will contribute the missing pipelines to the specific PRs today.

Reply:
Alright, I just rebased the PR to branch `users/itpropro/webpubsub`. 

I'll give the PR another review and once the pipelines are in, we can merge it into set branch. I will then go ahead, test the branch and create yet another PR to main.

As this, however, doesn't scale particulary well it would be awesome if you could setup your fork some time (if you have an Azure subscription available).

Reply:
I'll do the same for the other PRs of yours